roles/openshift_sanitize_inventory/tasks/__deprecations_metrics.yml

```diff
@@ -0,0 +1,27 @@
+---
+# this is used to set the metrics variables from deprecated values to the current variables names
+# this file should be deleted once variables are no longer honored
+
+- set_fact:
+    openshift_metrics_storage_access_modes: "{{ openshift_hosted_metrics_storage_access_modes | default(none) }}"
+    openshift_metrics_storage_host: "{{ openshift_hosted_metrics_storage_host | default(none) }}"
+    openshift_metrics_storage_nfs_directory: "{{ openshift_hosted_metrics_storage_nfs_directory | default(none) }}"
+    openshift_metrics_storage_volume_name: "{{ openshift_hosted_metrics_storage_volume_name | default(none) }}"
+    openshift_metrics_storage_volume_size: "{{ openshift_hosted_metrics_storage_volume_size | default(none) }}"
+    openshift_metrics_storage_labels: "{{ openshift_hosted_metrics_storage_labels | default(none) }}"
+
+- set_fact:
+    openshift_metrics_image_prefix: "{{ openshift_hosted_metrics_deployer_prefix }}"
+  when: openshift_hosted_metrics_deployer_prefix is defined
+
+- set_fact:
+    openshift_metrics_image_version: "{{ openshift_hosted_metrics_deployer_version }}"
+  when: openshift_hosted_metrics_deployer_version is defined
+
+- set_fact:
+    openshift_metrics_install_metrics: "{{ openshift_hosted_metrics_deploy }}"
+  when: openshift_hosted_metrics_deploy is defined
+
+- set_fact:
+    openshift_metrics_storage_kind: "{{ openshift_hosted_metrics_storage_kind }}"
+  when: openshift_hosted_metrics_storage_kind is defined
```

Comment:
This file seems unrelated to the topic of the PR?

Reply:
Ugh, i committed while on the wrong branch locally... but i thought i reverted it. Will remove